pavement.py

```diff
@@ -303,7 +303,7 @@ def bootstrap(options):
     bdir = options.bootstrap_dir
     if not os.path.exists(bdir):
         os.makedirs(bdir)
-    bscript = "boostrap.py"
+    bscript = "bootstrap.py"
```

Comment:
This changes the script name, which should make something that succeeded fail, or something that failed succeed

Reply:
I was going to ask about this specific typo. Thanks for catching it. Deferring to the devs